webpack/assets/javascripts/react_app/components/hosts/storage/vmware/StorageContainer.test.js

```diff
@@ -51,4 +51,18 @@ describe('StorageContainer', () => {
 
     expect(wrapper.render().find('.controller-container')).toHaveLength(0);
   });
+
+  it('changes controller type when one is selected', () => {
```

Comment:
you can use our TestHelpers instead of mounting components manually.
We already have helpers for unit-perspective  test (`TestHelpers.js`), redux action-reducers tests (`IntegrationTestHelper.js`)

Reply:
Thanks, I did not know that, I have updated the tests to use the helper.